### DISCUSSION TITLE
inject_assets:true not working in Lavravel 11 maintenance mode

### DISCUSSION THREAD

**Discussion by Ronan-Lenor** [livewire/livewire#8858] [7. Bugs (archived)]
### Livewire version

v3.5.6

### Laravel version

v11.23.5

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

Set the Maintenance mode `php artisan down`
Go to any page that use the layout (`503.blade.php`) > `resources\views\errors\minimal.blade.php`

Not only the layout require `@livewireStyles` & `@livewireScripts`.
But `livewire/livewire.js` also need to be added in `bootstrap\app.php`
```php
$middleware->preventRequestsDuringMaintenance(['stripe/*', 'livewire/livewire.js',])`
```

### Repository or Wirebox URL

none

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by lrljoe** [comment]
I mean, laravel maintenance mode is meant to disable your whole app, so this is expected?

Maintenance mode is designed so that you can perform upgrades to your server/app without any frontend updates taking place.

Forcing stuff to still work defeats the purpose of maintenance mode...

Or do you mean it doesn't work when using the bypass?
